kmip/services/server/kmip_server.py

```diff
@@ -70,9 +71,13 @@ def serve(self):
             try:
                 while True:
                     self._processor.process(protocol, protocol)
+            except EOFError as e:
+                self.logger.warning("KMIPServer {0} {1}".format(type(e), e))
             except Exception as e:
                 self.logger.error('KMIPServer {0} {1}'.format(type(e), e))
-                connection.close()
+
+            connection.close()
```

Comment:
Matter of style I suppose, but I prefer for `connection.close()` to be part of a `finally` block.